### DISCUSSION TITLE
Issue with video player, multiple video playing in background while no video playing in UI

### DISCUSSION THREAD

**Discussion by yashveertrigyn** [Sunbird-Ed/Community#659] [Bugs]
We are facing some issue with background video playing in sunbird Ed for courses.

For any course when we are clicking in continue learning button, all video of courses playing in background simultaneously(Not playing in UI). We commented code for video player then there is no background video playing in application. So we suspect somehow video playing in background automatically and even we go to any other module it still playing in background till we do not refresh browser.

Can you please let me know what and where logic is there to start the video playing and stopping the playing in case of a course and module consumption.

**One observation** : When we are using hard refresh for same screen then the error is not coming.

Attached screenshot and video for reference. 

https://github.com/Sunbird-Ed/Community/assets/141142203/a6a20f74-4ff4-4691-8282-ab91ed3a31af

[
![countinue-button](https://github.com/Sunbird-Ed/Community/assets/141142203/540eb8fd-0a9d-4ae9-bf94-aaa6c3436aa0)
](url)

**Comment by heungheung** [comment]
From the screen capture, I saw the course is 100% completed. Is this only for course with 100% completion?  

What if course is, e.g. 50% completed? E.g. in 50% completed case, when user continue and being redirected to when she/he should get started, will
- all media in the course will be played at the same time in the background? or
- all media for the completed 50% will be played at the same time in the background?

**Comment by yashveertrigyn** [comment]
it is happening for all cases when clicking in continue learning button. Below are the cases.

Steps:
Scenario 1
1.open Diksha portal
2.Login with the custodian user credentials
3.search for the course with video
4.Join the course
5. Complete the course
6. Click on back and go the course toc page
7.Click on continue learning 

scenario 2
1.open Diksha portal
2.Login with the custodian user credentials
3. search for the course with video
4.Join the course
5.Consume the Mp4 content 
6. Click on the next module and complete the module 
7. Click on the previous module

**Comment by yashveertrigyn** [comment]
For 100% completed courses. if we are clicking any courses from right sidebar. it is happening sometime.

**Comment by yashveertrigyn** [comment]
We tried @heungheung suggestion but it is not working. Facing same issue.

**Reply by heungheung** [reply]
Are you talking about autoplay?

**Reply by yashveertrigyn** [reply]
yes.
